src/libsyntax/error_codes.rs

```diff
@@ -432,7 +432,7 @@ register_diagnostics! {
     E0544, // multiple stability levels
     E0545, // incorrect 'issue'
     E0546, // missing 'feature'
-    E0547, // missing 'issue'
+    E0547, // missing 'issue' //TODO I reused this one for the warning.
```

Comment:
Is that a good idea? :/

Reply:
No it's not, but I don't know how to properly do it. Can someone help me here?